### PR TITLE
Do not use auto_expand_replicas

### PR DESCRIPTION
**Pull request by lukas-vlcek**
Back porting https://github.com/openshift/openshift-ansible/pull/3580 to release-1.5
//cc @ewolinetz @richm

### CODE REVIEW

**Comment by lukas-vlcek** [comment]
Trying to fix conflicts...

File: roles/openshift_logging/tasks/install_elasticsearch.yaml [hunk 1]
```diff
@@ -44,7 +44,10 @@
     volume_names: "{{es_pvc_pool | default([])}}"
     pvc_claim: "{{(volume_names | length > item.0) | ternary(volume_names[item.0], None)}}"
     deploy_name: "{{item.1}}"
-    es_node_selector: "{{openshift_logging_es_nodeselector | default({})}}"
+    es_node_selector: "{{openshift_logging_es_nodeselector | default({}) }}"
+    es_storage: "{{openshift_logging_facts|es_storage(deploy_name, pvc_claim)}}"
```
Comment (richm) [inline roles/openshift_logging/tasks/install_elasticsearch.yaml]: What is this es_storage?

File: roles/openshift_logging/tasks/install_elasticsearch.yaml [hunk 2]
```diff
@@ -44,7 +44,10 @@
     volume_names: "{{es_pvc_pool | default([])}}"
     pvc_claim: "{{(volume_names | length > item.0) | ternary(volume_names[item.0], None)}}"
     deploy_name: "{{item.1}}"
-    es_node_selector: "{{openshift_logging_es_nodeselector | default({})}}"
+    es_node_selector: "{{openshift_logging_es_nodeselector | default({}) }}"
```
Comment (richm) [inline roles/openshift_logging/tasks/install_elasticsearch.yaml]: This node selector change seems unrelated

Comment (lukas-vlcek) [inline roles/openshift_logging/tasks/install_elasticsearch.yaml]: There are conflicts... fixing it now.

**Comment by lukas-vlcek** [comment]
@ewolinetz @richm Sorry, I messed cherry-picking my PR to `release-1.5` branch previously. Just updated it now. Do you think you can have a look?

**Comment by ewolinetz** [comment]
[test]
